sql-statements/sql-statement-show-profiles.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Please add this file link `/sql-statements/sql-statement-show-profiles.md` to TOC.md